guide/using-html-in-markdown.md

```diff
@@ -0,0 +1,83 @@
+# Using HTML in Markdown, in Neuron
+
+Usually, Markdown supports raw HTML inline as valid Markdown syntax.
```

Comment:
Since neuron uses `commonmark` (which does support raw HTML), you should link to the spec: https://spec.commonmark.org/0.28/#raw-html